eclass/emboss.eclass

```diff
@@ -97,8 +94,19 @@ DOCS="AUTHORS ChangeLog NEWS README"
 #
 
 emboss_src_prepare() {
+	[[ -n ${EBO_PATCH} ]] && epatch "${WORKDIR}"/${P}-upstream.patch
 	[[ -f ${FILESDIR}/${PF}.patch ]] && epatch "${FILESDIR}"/${PF}.patch
-	[[ ${EBO_EAUTORECONF} == yes ]] && eautoreconf
+
+	# Delete pointless install-exec-hook update check
+        sed -i -e '/install-exec-hook:/,+1d' Makefile.am
+
+	if [[ ${EBO_EAUTORECONF} == yes ]];
+	then
+		AUTOTOOLS_AUTORECONF=1
```

Comment:
this was done more as a wrapper in order to retain ${EBO_EAUTORECONF}, which given a choice of ${AUTOTOOLS_AUTORECONF} should be removed.